File: output/iter-R20-pytest/course/lesson01/in_class/practice02.py
"""
[难度: ★][所属知识点: fixture][预计完成时间: 5 分钟]

题目:用 fixture 提供测试数据 [1, 2, 3, 4, 5],测试 sum 和 len。
"""

if __name__ == "__main__":
    print("参考答案:")
    print("  @pytest.fixture")
    print("  def data(): return [1, 2, 3, 4, 5]")
    print("  def test_sum(data): assert sum(data) == 15")
    print("  def test_len(data): assert len(data) == 5")

File: output/iter-R9-js-build-pedagogy/course/lesson03/in_class/practice02.js
/*
[难度: ★]
[所属知识点: 场景选择(简单)]
[预计完成时间: 5 分钟]

题目:给定以下场景,选择 let/const 并说明理由。

    1. 圆周率 PI = 3.14159
    2. 累加器 sum,每次循环加一个数
    3. 用户列表 users,声明后需要 push 新用户
    4. 循环计数器 i
*/

if (require.main === module) {
  console.log("1. PI → const(永远不变)");
  console.log("2. sum → let(需要重新赋值)");
  console.log("3. users → const(对象本身不变,内容可变)");
  console.log("4. i → let(块级,循环结束后释放)");
}
